cdispyutils/auth/jwt_validation.py

```diff
@@ -173,6 +173,8 @@ def validate_request_jwt(
     request = request or flask.request
     try:
         encoded_token = request.headers['Authorization'].split(' ')[1]
+    except IndexError:
```

Comment:
Catch all exceptions when parsing auth headers? Or does validate_jwt do that?

Reply:
Also when you you say the word edge case and don't add a test case : http://i0.kym-cdn.com/photos/images/facebook/000/956/638/5bc.gif

Reply:
That statement specifically should only be able to raise `IndexError` or `KeyError` as far as I can tell. `validate_jwt` handles everything else, basically.

Thanks for pointing that out! I added a regression test for this, but in fence, not here…will add.

Reply:
Test case added in 866304c.